classes/classes/Items/MutationsHelper.as

```diff
@@ -10,39 +10,65 @@ package classes.Items
 	{
 		include "../../../includes/appearanceDefs.as";
 
+		// I tend to use bitfields rather than lots of optional boolean params.
+		// If I consider a method to be finalized and has only one option I'll refactor this to use a boolean value.
+		// I'll add more consts later if needed. This one is just here for the sake of being an example for starters (Stadler)
+
+		// restoreArms options
+		public static const RESTOREARMS_FROMGOOSKINTF:int = 1;
```

Comment:
This flag-const is very lonely. I'll add more friends for it later, when I'm improving that class more and more.